cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -12,35 +12,40 @@
 
    ========================================================================== #}
 
+{% if value.data  %}
 <table class="o-table
               {{ 'o-table__row-links' if value.row_links else '' }}
-              table__stack-on-small
-              table__entry-header-on-small">
+              {{ 'table__stack-on-small' if value.is_stacked else '' }}
+              {{ 'table__striped' if value.is_striped else '' }}
+              {{ 'u-w100pct' if value.is_full_width else '' }}">
+    {% set row_index = 0 %}
+    {% if value.first_row_is_table_header and value.data | list | length > 0 %}
+    {% set row_index = 1 %}
     <thead>
         <tr>
-            {% for thead in value.headers %}
-                <th>{{ thead }}</th>
+            {% for column in value.data[0] %}
+                {% if cell is not none %}
+                <th>
+                    {{ column | trim | safe | linebreaksbr }}
+                </th>
+                {% endif %}
             {% endfor %}
         </tr>
     </thead>
-    {% for row in value.rows %}
+    {% endif %}
+    <tbody>
+    {% for row in value.data[row_index:] or [] %}
         <tr>
-        {% for cell in row %}
-            {% set colcount = loop.index %}
-            {% for thead in value.headers %}
-                {% if colcount == loop.index %}
-                    {% if 'rich_text_blob' in cell.block_type %}
-                        <td data-label="{{ thead }}">
-                            {{ parse_links(cell.value) | safe }}
-                        </td>
-                    {% else %}
-                        <td data-label="{{ thead }}">
-                            {{ cell | safe }}
-                        </td>
-                    {% endif %}
+            {% for cell in row or [] %}
+                {% if cell is not none %}
```

Comment:
This fixes @chosak's [comment](https://github.com/cfpb/cfgov-refresh/pull/2344#issuecomment-243822904) but now if you leave a cell empty, it's removed and the cells to the right are shifted to the left. See screenshots below. Would it be possible to insert `&nbsp;` or some sort of invisible placeholder to preserve empty cells?

![screen shot 2016-09-21 at 4 54 45 pm](https://cloud.githubusercontent.com/assets/1060248/18728736/9d3ce782-801c-11e6-82cc-1e334a5c9a1a.png)

![screen shot 2016-09-21 at 4 54 21 pm](https://cloud.githubusercontent.com/assets/1060248/18728744/a29b164a-801c-11e6-839a-2941f79705d8.png)